src/utils/resolveYextEntityField.ts

```diff
@@ -14,7 +14,7 @@ export const resolveYextEntityField = <T>(
   }
 
   // return constant value if field is not set
-  if (entityField.field === "") {
+  if (entityField.field === "" || entityField.constantValueEnabled) {
```

Comment:
We should only render constant value if it's enabled now. If field is blank then it should just return empty string along with a console warning. Though I'm not 100% sure about that. Let's check with Aaron on the fallback. But IMO if you haven't enabled constant value then it should never use it.

Reply:
Okay! I'll ask

Reply:
Conclusion, don't fallback. https://yext.slack.com/archives/C06A06BCUUF/p1727961313799639?thread_ts=1727957859.737239&cid=C06A06BCUUF